src/FileServiceManager/FileService/Helpers/FileExtension.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace FileService.Helpers
+{
+    public class FileExtension
+    {
+        public string Name { get; set; }
+
+        public byte[][] Signatures { get; set; }
```

Comment:
What is signature ?

Reply:
Signature is a byte array, that every file has as its head bytes. As user can rename file with extension .exe to .txt and try to upload it, it is better to check file extension by its signature. Every file extension has different signatures, associated with it. More on https://www.filesignatures.net/
I've updated appsettings.json to show an example of what signature can be.